.pipelines/pipeline.yaml

```diff
@@ -153,6 +153,14 @@ stages:
                 arch: amd64
                 name: npm
                 os: windows
+              cni_telemetry_sidecar_linux_amd64:
+                arch: amd64
+                name: cni-telemetry-sidecar
+                os: linux
+              cni_telemetry_sidecar_windows_amd64:
+                arch: amd64
+                name: cni-telemetry-sidecar
+                os: windows
```

Comment:
instead of publishing a separate image, can we just build this in to the azure-cni image and run it from there?